cudax/test/execution/test_stream_context.cu

```diff
@@ -8,13 +8,16 @@
 //
 //===----------------------------------------------------------------------===//
 
+// BUGBUG BUGBUG
```

Comment:
What's the bug?

Reply:
the bug is that i forgot to remove this comment. :-P